### PR TITLE
docs(v-autocomplete): slot descriptions

### PR DESCRIPTION
**Pull request by mohkermani**
fix doc v-autocomplete slots

## Markup:

```vue

```

### CODE REVIEW

**Comment by mohkermani** [comment]
improve doc v-autocomplete slotd

File: packages/api-generator/src/locale/en/v-autocomplete.json [hunk 1]
```diff
@@ -23,12 +23,12 @@
...
     "clear": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/next/packages/api-generator/src/locale/en/v-autocomplete.json))",
-    "prepend-inner": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/next/packages/api-generator/src/locale/en/v-autocomplete.json))",
-    "append-inner": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/next/packages/api-generator/src/locale/en/v-autocomplete.json))",
-    "loader": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/next/packages/api-generator/src/locale/en/v-autocomplete.json))",
-    "item": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/next/packages/api-generator/src/locale/en/v-autocomplete.json))",
-    "chip": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/next/packages/api-generator/src/locale/en/v-autocomplete.json))",
-    "selection": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/next/packages/api-generator/src/locale/en/v-autocomplete.json))",
+    "prepend-inner": "Appends slot outside of input",
```
Comment (MajesticPotatoe) [inline packages/api-generator/src/locale/en/v-autocomplete.json:26]:
```suggestion
    "prepend-inner": "Slot to insert items inside the input and before input content",
```

File: packages/api-generator/src/locale/en/v-autocomplete.json [hunk 2]
```diff
@@ -23,12 +23,12 @@
...
+    "append-inner": "Appends slot inside input",
```
Comment (MajesticPotatoe) [inline packages/api-generator/src/locale/en/v-autocomplete.json:27]:
```suggestion
    "append-inner": "Slot to insert items inside the input and after input content",
```

**Review by MajesticPotatoe — CHANGES_REQUESTED** [review]
Some other quick easy wins:
- prepend/append and prepend/append-item (same verbiage as prepend/append-inner)
- clear is just slot for `clearable` prop

File: packages/api-generator/src/locale/en/v-autocomplete.json [hunk 3]
```diff
@@ -6,31 +6,31 @@
...
+    "clear": "slot for clear",
+    "prepend-inner": "Prepends slot inside of input",
+    "append-inner": "Appends slot inside of input",
+    "loader": "Defines loaders appearance of the autocomplete (useful when you fetch data from server)",
+    "item": "Defines custom appearance of items in select when menu is open for selection of autocomplete value",
+    "chip": "Defines custom appearance of the selected item(s) whens using chips",
+    "selection": "Defines custom appearance of the selected item in autocomplete input",
+    "prepend-item": "prepends item",
+    "append-item": "Appends to item",
+    "no-data": "Slot shiwn when there is no autocomplete data"
```
Comment (MajesticPotatoe) [inline packages/api-generator/src/locale/en/v-autocomplete.json:33]:
```suggestion
    "no-data": "Slot shown when there is no autocomplete data"
```

File: packages/api-generator/src/locale/en/v-autocomplete.json [hunk 4]
```diff
@@ -6,31 +6,31 @@
...
-    "chip": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/master/packages/api-generator/src/locale/en/v-autocomplete.json))",
-    "selection": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/master/packages/api-generator/src/locale/en/v-autocomplete.json))",
-    "prepend-item": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/master/packages/api-generator/src/locale/en/v-autocomplete.json))",
-    "append-item": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/master/packages/api-generator/src/locale/en/v-autocomplete.json))",
-    "no-data": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/master/packages/api-generator/src/locale/en/v-autocomplete.json))"
+    "prepend": "Prepends slot outside of input",
+    "append": "Appends slot outside of input ",
+    "details": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/next/packages/api-generator/src/locale/en/v-autocomplete.json))",
```
Comment (MajesticPotatoe) [inline packages/api-generator/src/locale/en/v-autocomplete.json:23]:
```suggestion
    "details": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/master/packages/api-generator/src/locale/en/v-autocomplete.json))",
```

**Review by MajesticPotatoe — CHANGES_REQUESTED** [review]

**Comment by johnleider** [comment]
These changes should occur in the components autocomplete is inheriting from:

image

**Comment by MajesticPotatoe** [comment]
@mohkermani  pulse check

**Comment by mohkermani** [comment]
> @mohkermani pulse check
> @johnleider These changes should occur in the components autocomplete is inheriting from:

Actually I cannot find any of slot descriptions in main file for v-input. there's only detail.

/vuetify/packages/api-generator/src/locale/en/v-input.json :
```
...
  "slots": {
    "details": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/master/packages/api-generator/src/locale/en/v-input.json))"
  },
...
```

**Comment by MajesticPotatoe** [comment]
> Actually I cannot find any of slot descriptions in main file for v-input. there's only detail.

locale files were renamed on `master`, v-input.json is now VInput.json

**Comment by mohkermani** [comment]
> > Actually I cannot find any of slot descriptions in main file for v-input. there's only detail.
> 
> locale files were renamed on `master`, v-input.json is now VInput.json

@MajesticPotatoe It gets more complicated :D There's only "props", "events", and "exposed" sections in the master VInput.
[ new VInput](https://github.com/vuetifyjs/vuetify/blob/master/packages/api-generator/src/locale/en/VInput.json)

**Comment by MajesticPotatoe** [comment]
> > > Actually I cannot find any of slot descriptions in main file for v-input. there's only detail.
> > 
> > 
> > locale files were renamed on `master`, v-input.json is now VInput.json
> 
> @MajesticPotatoe It gets more complicated :D There's only "props", "events", and "exposed" sections in the master VInput. [ new VInput](https://github.com/vuetifyjs/vuetify/blob/master/packages/api-generator/src/locale/en/VInput.json)

Then add it?
(I would fix merge conflicts first though before doing anything)

**Pull request closed without merging**